### PR TITLE
Kate/93332/CD: [Change Request] replace contract card fields by new ones

### PR DESCRIPTION
**Pull request by kate-deriv**
## Changes:

- In `Contract card`: replace old values and add new one (buy price, contract value, entry spot, take profit and barrier)
- In `Details`: add `Duration`, `Barrier` and `Payout per point` for `Turbos` contract (both for unsold and sold)
- Change tests accordingly 

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by maryia-deriv — APPROVED** [review]
👍🏻

File: packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx [hunk 1]
```diff
@@ -54,72 +54,64 @@ const TurbosCardBody = ({
...
-                    is_loss={is_sold && profit ? profit < 0 : false}
-                    is_won={is_sold && profit ? profit > 0 : false}
+                    header={PURCHASE_PRICE}
                 >
-                    <Money amount={is_sold ? profit : buy_price} currency={currency} />
+                    <Money amount={buy_price} currency={currency} />
                 </ContractCardItem>
-                <ContractCardItem header={is_sold ? PAYOUT : CURRENT_PRICE} className='dc-contract-card__current-price'>
-                    <Money currency={currency} amount={sell_spot || current_spot_display_value} />
+                <ContractCardItem header={CONTRACT_VALUE} className='dc-contract-card__current-value'>
```
Comment (maryia-deriv) [inline packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx]:
```suggestion
                <ContractCardItem header={CONTRACT_VALUE} className='dc-contract-card__contract-value'>
```
nit😊

**Review by maryia-deriv — APPROVED** [review]
👍🏻

File: packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx (hunk 1 above)
Comment (kate-deriv) [reply packages/components/src/components/contract-card/contract-card-items/turbos-card-body.tsx]: Done🫡

**Pull request merged**